pkg/ims/service/service.go

```diff
@@ -140,4 +148,34 @@ func (service *SignUpService) EmailVerificationToken (token *models.EmailVerific
 	return nil
 }
 
-// Send Email Verification Token function must be here!
\ No newline at end of file
+// Send Email Verification Token function must be here!
+func (service *CredService) ChangePassword(request *models.Credentials) error {
+
+	var err error
+	var user models.User
+
+	tx, err := service.db.Beginx()
+	if err != nil {
+		return err
+	}
+
+	defer tx.Rollback()
+
+	credRepo := postgresql.NewCredentialsRepository(tx)
+
+	request.ID = user.ID
+
+	err = credRepo.UpdateCredentials(request)
+
```

Comment:
Remove the indent between the error declaration and error checking, i think it will make the code more readable and in the same style.